src/models/vehicle/Chassis.jsx

```diff
@@ -24,12 +24,13 @@ const Chassis = forwardRef(({ args = [2, 1.1, 4.7], mass = 500, children, ...pro
   const ready = useStore((state) => state.ready)
   const { nodes: n, materials: m } = useGLTF('/models/chassis-draco.glb')
 
-  const onCollide = useCallback(
+  const onCollide = () => {
+    if (!useStore.getState().controls.sfx) return
     debounce((e) => {
-      crashAudio.current?.setVolume(clamp(e.contact.impactVelocity / 10, 0.2, 1))
+      crashAudio.current?.setVolume((0.4 * useStore.getState().speed) / 10)
```

Comment:
was this a bad merge? not sure why this has changed

Reply:
oh that's weird. Yes that should be reverted.

Reply:
i think after that we can merge

Reply:
i'll fix it